tic_tac_toe/game.py

```diff
@@ -24,17 +26,28 @@ def _position_is_valid(position):
 
     Returns True if given position is valid, False otherwise.
     """
-    pass
-
+    if not isinstance(position, tuple):
+        return False
+    if len(position) != 2:
+        return False
+    if position[0] not in [0,1,2]:
```

Comment:
is the point of the `row, col = position` just clarity? I don't disagree, just trying to understand!